Modules/pyexpat.c

```diff
@@ -581,7 +592,7 @@ my_ElementDeclHandler(void *userData,
 
         if (flush_character_buffer(self) < 0)
             goto finally;
-        modelobj = conv_content_model(model, (conv_string_to_unicode));
+        modelobj = conv_content_model(model, (conv_string_to_unicode_void));
```

Comment:
Are the `()` around `conv_string_to_unicode_void` necessary?

Reply:
It seems like they can be removed.